mmv1/products/chronicle/Watchlist.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Watchlist
+description: A watchlist is a list of entities that allows for bulk operations over the included entities.
+min_version: beta
+references:
+  guides:
+    'Google SecOps Guides': 'https://cloud.google.com/chronicle/docs/secops/secops-overview'
+  api: 'https://cloud.google.com/chronicle/docs/reference/rest/v1alpha/projects.locations.instances.watchlists'
+base_url: projects/{{project}}/locations/{{location}}/instances/{{instance}}/watchlists
+self_link: projects/{{project}}/locations/{{location}}/instances/{{instance}}/watchlists/{{watchlist_id}}
+create_url: projects/{{project}}/locations/{{location}}/instances/{{instance}}/watchlists?watchlistId={{watchlist_id}}
+id_format: projects/{{project}}/locations/{{location}}/instances/{{instance}}/watchlists/{{watchlist_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/instances/{{instance}}/watchlists/{{watchlist_id}}
+update_verb: PATCH
+update_mask: true
+
+examples:
+  - name: 'chronicle_watchlist_basic'
+    primary_resource_id: 'example'
+    min_version: 'beta'
+    vars:
+      name: watchlist-name
+      description: watchlist-description
+    test_env_vars:
+      chronicle_id: 'CHRONICLE_ID'
+
+parameters:
+  - name: location
+    type: String
+    description: Resource ID segment making up resource `name`. It identifies the resource within its parent collection as described in https://google.aip.dev/122.
+    immutable: true
+    url_param_only: true
+    required: true
+  - name: instance
+    type: String
+    description: Resource ID segment making up resource `name`. It identifies the resource within its parent collection as described in https://google.aip.dev/122.
+    immutable: true
+    url_param_only: true
+    required: true
+  - name: watchlistId
+    type: String
+    description: |-
+      Optional. The ID to use for the watchlist,
+      which will become the final component of the watchlist's resource name.
+      This value should be 4-63 characters, and valid characters
+      are /a-z-/.
+    immutable: true
+    url_param_only: true
+    required: true
+properties:
+  - name: name
+    type: String
+    description: |-
+      Identifier. Resource name of the watchlist.
+      Format:
+      projects/{project}/locations/{location}/instances/{instance}/watchlists/{watchlist}
+    default_from_api: true
```

Comment:
This will always be formed by the other fields like project, location, instance and watchlist, and will never be provided by the user. So as per your suggestion, I believe it will be better to use `output: true` instead. Please correct me if I am wrong.

Reply:
Then `output: true` makes sense here.

Reply:
Made the change, thanks for the suggestion.